data/src/audio.rs

```diff
@@ -0,0 +1,60 @@
+use std::path::PathBuf;
+
+use serde::Deserialize;
+
+use crate::Config;
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum Sound {
+    Internal(Internal),
+    External(String),
+    None,
+}
+
```

Comment:
Instead of encoding None here, we can do `Option<Sound>` which is more idiomatic.

Also, using Option<T> with serde works ootb and you won't need to define a custom default definition.